### PR TITLE
Added Ten More Components For Library

### PR DESCRIPTION
**Pull request by gooddavvy**
I added ten more components, dad. We'll, one of them I issued to you, but the rest nine. Working well.

### CODE REVIEW

File: chartsLibrary/SolidSparklineChart.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { onMount } from "solid-js";
+import { AreaChart } from "@carbon/charts";
+import { ChartConfig, AreaChartOptions } from "@carbon/charts/interfaces";
+
+let divRef: HTMLDivElement = (<div></div>) as HTMLDivElement;
+
+export default function SolidLineChart(props: ChartConfig<AreaChartOptions>) {
```
Comment (asolpshinning) [inline chartsLibrary/SolidSparklineChart.tsx]: This function is AreaLineChart... does not really matter. But please correct it

File: chartsLibrary/SolidStepChart.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
+import { onMount } from "solid-js";
+import { LineChart } from "@carbon/charts";
+import { ChartConfig, LineChartOptions } from "@carbon/charts/interfaces";
+
+const data = [
```
Comment (asolpshinning) [inline chartsLibrary/SolidStepChart.tsx:5]: No data should be in library components

File: examples/PieChartExample.tsx [hunk 3]
```diff
@@ -2,46 +2,142 @@ import DonutChart from "../chartsLibrary/SolidDonutChart";
 import SolidPieChart from "../chartsLibrary/SolidPieChart";
 
 const data = [
```
Comment (asolpshinning) [inline examples/PieChartExample.tsx:4]: Please restore what was in this component. This was not supposed to be edited.

**Review by asolpshinning — CHANGES_REQUESTED** [review]
Great job!!! Please address all my comments and I will merge this.

**Comment by gooddavvy** [comment]
Thanks, all you referred to here will be fixed, and I'll add some more charts for the library.
Good luck out there!

File: chartsLibrary/SolidSparklineChart.tsx (hunk 1 above)
Comment (gooddavvy) [reply chartsLibrary/SolidSparklineChart.tsx]: ok

File: chartsLibrary/SolidStepChart.tsx (hunk 2 above)
Comment (gooddavvy) [reply chartsLibrary/SolidStepChart.tsx:5]: That was just a mistake, dad. Just a mistake.

File: examples/PieChartExample.tsx (hunk 3 above)
Comment (gooddavvy) [reply examples/PieChartExample.tsx:4]: I did not edit anything here, dad. This was the same in my VS code.

**Pull request closed without merging**